app/mailers/user_mailer.rb

```diff
@@ -1,4 +1,6 @@
 class UserMailer < AsyncMailer
+  add_template_helper WritableHelper
```

Comment:
(apparently this is getting removed in Rails 6.1 and should be done with `helper WritableHelper` in the future? I wonder if that already works for us - https://stackoverflow.com/questions/65528227/the-method-add-template-helper-does-not-seem-to-work-in-rails-6-1)